client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -220,46 +202,59 @@ const Security2faSMSSettings = createReactClass( {
 					<FormButtonsBar className="security-2fa-sms-settings__buttons">
 						<FormButton
 							disabled={ this.getSubmitDisabled() }
-							onClick={ function ( event ) {
+							onClick={ ( event ) => {
```

Comment:
I wonder if we should declare this as a component method so we won't have to redeclare them on every render.